demo-projects/meetup/schema.js

```diff
@@ -109,6 +109,38 @@ exports.Rsvp = {
     user: { type: Relationship, ref: 'User' },
     status: { type: Select, options: 'yes, no' },
   },
+  hooks: {
+    validateInput: async ({ resolvedData, existingItem, actions }) => {
+      const { status } = resolvedData;
+      const { event: eventId } = existingItem ? existingItem : resolvedData;
+
+      if (status === 'no') {
+        return;
+      }
+
+      const { data } = await actions.query(`query {
+        event: Event(where: { id: "${eventId}" }) {
+          id
+          startTime
+          maxRsvps
+        }
+        allRsvps(where: { event: { id: "${eventId}" }}) {
+          id
+        }
+      }`);
+
+      const { event, allRsvps } = data;
+
+      if (
+        !event ||
+        !event.startTime ||
+        new Date() > event.startTime ||
```

Comment:
This comparison would be a timestamp vs a string, should the right hand side be wrapped in a `new Date`?